### PR TITLE
Load the dynamic committee members data from firebase 

### LINKED ISSUE #30: GOSA11 - Load the dynamic committee members data from firebase
Fetch data from firebase and display in the app (UI for this is already done).

### PR DESCRIPTION
**Pull request by singhtaran1005**
GOSA11
ISSUE #30 

#### Describe the changes you've made
As told I had made collections along with doc id with respective fields as mentioned in UI for committee members.I had also created one individual member list where i have added details of all members at once and called them in the members.dart file under screens folder through which they can be loaded from Firebase.For now i have added name description and designation in fields of Firebase which can be edited accordingly and for one example i have loaded one example image in member 5.All members are in a list view.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Additional context (OPTIONAL)
Screenshots-
WhatsApp Image 2020-10-24 at 12 34 34 PM

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/GameofSource-GFG/Web-Development/blob/main/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by singhtaran1005** [comment]
For now i have just kept 6 fields in documents under members collection in firebase

**Review by VarunSAthreya — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts.
After that PR will be merged

**Comment by amanv8060** [comment]
@VarunSAthreya  These Merge Conflict can be resolved from github itself . Merge conflict has occurred due to structural change in code .

**Comment by singhtaran1005** [comment]
okay resolved conflicts!!

**Comment by VarunAthreya** [comment]
> okay resolved conflicts!!

error

This is the error!

File: lib/models/individual_member.dart [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                mainAxisAlignment: MainAxisAlignment.start,
+                mainAxisSize: MainAxisSize.min,
+                children: <Widget>[
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisSize: MainAxisSize.min,
+                    children: <Widget>[
+                      Text(
+                        member.speakerName,
+                        style: Theme.of(context).textTheme.title,
```
Comment (amanv8060) [inline lib/models/individual_member.dart]: Replace title with headline6

**Comment by singhtaran1005** [comment]
actually the thing is if you use headline the speaker name will come a big larger

**Comment by singhtaran1005** [comment]
still doing changes

**Review by VarunSAthreya — APPROVED** [review]
Congratulations on the successful merge!
The next set of issues would be out soon.

**Pull request merged**